src/utils/dappMailer.js

```diff
@@ -243,9 +243,9 @@ module.exports = {
       title: 'Milestone review requested',
       image: 'Giveth-review-banner-email.png',
       text: `
-        <p><span style="line-height: 33px; font-size: 22px;">Hi ${data.user}</span></p>
+        <p><span style="line-height: 33px; font-size: 22px;">Hi ${data.user(0).toUpperCase()+data.user.slice(1)}</span></p>
```

Comment:
user is of type string, so user(0) doesn't have any meaning and creates exception!